homeassistant/const.py

```diff
@@ -309,9 +309,9 @@ class Platform(StrEnum):
 EVENT_PANELS_UPDATED: Final = "panels_updated"
 EVENT_LOVELACE_UPDATED: Final = "lovelace_updated"
 EVENT_RECORDER_5MIN_STATISTICS_GENERATED: Final = "recorder_5min_statistics_generated"
-EVENT_RECORDER_HOURLY_STATISTICS_GENERATED: Final = (
-    "recorder_hourly_statistics_generated"
-)
+EVENT_RECORDER_HOURLY_STATISTICS_GENERATED: (
+    Final
+) = "recorder_hourly_statistics_generated"
```

Comment:
What happened here, why is it changed?

Reply:
This was auto-changed by the formatter, I use the Dev Container...